cmake/README.md

```diff
@@ -0,0 +1,118 @@
+This directory contains cmake files that can be used to generate MSVC project
+files in order to build protobuf on windows. You need to have cmake installed
+on your computer before proceeding.
+
+Compiling and Installing
+========================
+
+0) Check whether a gtest directory exists in the upper level directory. If
+   you checkout the code from github via "git clone", this gtest directory
+   won't exist and you won't be able to build the tests described below. To
+   avoid this problem consider downloading one of the release tar balls which
+   contains gtest already and copying the gest directory from there to your
+   protobuf directory:
+       https://github.com/google/protobuf/releases
+1) Use cmake to generate MSVC project files. Running the following commands
+   in a command shell will generate project files for Visual Studio 2008 in
+   a sub-directory named "build".
+     > cd path/to/protobuf/cmake
+     > mkdir build
+     > cd build
+     > cmake -G "Visual Studio 9 2008" ..
```

Comment:
It looks like text after '..' becomes grey for some reason. Please double check if it is some special mark down syntax.